### ISSUE TITLE
expand use of solvers beyond Python

### ISSUE THREAD

**Issue by mihai-sysbio** [cdanielmachado/solver_benchmark#1]
Great work, and very much needed 🎉  I was super happy to see this!

I'd be curious to see how the solvers perform if used via other programming languages (e.g. Julia via [Cobrexa](https://github.com/LCSB-BioCore/COBREXA.jl) and Matlab via Cobra toolbox or Raven).

**Comment by cdanielmachado** [comment]
I think all the solvers are implemented in C++ and offer APIs for multiple languages. Calling the APIs via some interface creates some overhead, but this should be somewhat constant. The time it takes to solve the problem is what increases with problem size, and I don't think it will be affected by the choice of interface.

**Comment by mihai-sysbio** [comment]
Indeed, it would make sense to see that the solvers perform equally irrespective of framework to the point that it would be a huge shock if otherwise.

Coming back to Cobrexa - via JuMP, there are [more solvers](https://jump.dev/JuMP.jl/stable/installation/#Supported-solvers) (some were already looked at).
